addons/base_vat/i18n/base_vat.pot

```diff
@@ -53,8 +53,8 @@ msgstr ""
 #. module: base_vat
 #: model_terms:ir.ui.view,arch_db:base_vat.res_config_settings_view_form
 msgid ""
-"If this checkbox is ticked, you will not be able to save a contact if its "
-"VAT number cannot be verified by the European VIES service."
+"If this checkbox is ticked, the default fiscal position that applies will "
+"depend upon the output of the verification by the European VIES Service."
```

Comment:
in my opinion, since this is in stable, it would be better to add a new translation instead of changing the existing one, but I might be wrong?

Reply:
It's preferable to break the old translation (since it is wrong enough as to be misleading)